storage/tokudb/hatoku_hton.cc

```diff
@@ -274,10 +288,27 @@ static int tokudb_init_func(void *p) {
     int r;
 
     // 3938: lock the handlerton's initialized status flag for writing
-    tokudb_hton_initialized_lock.lock_write();
+    //
+    // tokudb_hton_initialized_lock can not be instrumented as it must be
```

Comment:
A better place for this comment is mutex declaration/definition/create/init rather than lock (the locking operation itself does not care whether it's instrumented or not)